en/docs/manage-apis/deploy-and-publish/deploy-on-gateway/federated-gateways/deploy-on-aws-api-gateway.md

```diff
@@ -30,53 +30,72 @@ Follow the instructions given below to configure AWS API Gateway as a Federated
+## Step 5 : Configure Security for the API
 
-    !!!note
-        Please note that only REST APIs are supported for deployment to AWS API Gateway.
+AWS APIs are secured using AWS OAuth2 policy. The policy takes in a `Lambda ARN` and a `Lambda Invoke Role ARN` as parameters. These information will be used to configure the Lambda Authorizer at AWS Gateway. You can implement the Lambda function to validate the tokens from an IDP of your choice. 
 
-## Step 4 : Configure Security for the API
+When the AWS API Gateway receives a request to a secured API, it will pass the request through the relevant Authorizer function before directing the request to the actual backend API.
+Please follow the steps below to configure the security for the API.
 
-AWS APIs are secured using AWS OAuth2 policy. The policy takes in a `Lambda ARN` and a `Lambda Invoke Role ARN` as parameters. These information will be used to configure the Lambda Authorizer at AWS Gateway. You can implement the Lambda function to validate the tokens from an IDP of your choice. Please follow the steps below to configure the security for the API.
+!!!note
+    Below steps would require additional permissions to be granted to the IAM user created in Step 1. Attach `AWSLambda_FullAccess` permission to the user and attach below custom policy to allow sts:AssumeRole action.
+    ```
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+            "Sid": "test",
+            "Effect": "Allow",
+            "Action": [
+               "iam:GetRole",
+               "iam:PassRole"
+            ],
+            "Resource": "*"
+            }
+        ]
+    }
+    ```
 
-??? note "Configuring Lambda Function and Invoke Role"
-    1. Configure a Lambda function to validate the OAuth2 tokens issued by the IDP of your choice.    
-    2. Configure a Custom Role with **lambda:InvokeFunction** permission to be used as the Invoke Role of the above Lambda. A sample permission JSON is shown below.
-        ``` 
-        {
-            "Version": "2012-10-17",
-            "Statement": [
-                {
-                    "Effect": "Allow",
-                    "Action": [
-                        "lambda:InvokeFunction"
-                    ],
-                    "Resource": [
-                        "*"
-                    ]
-                }
-            ]
-        }
-        ```
-    3. Also configure a trust relationship for the role to allow **sts:AssumeRole**.
+1. Configure a Lambda function in AWS to validate tokens generated from the IDP configured in step 3. You may refer to the sample lambda function implementation at [https://github.com/wso2/samples-apim/tree/master/custom-lambda-authorizer](https://github.com/wso2/samples-apim/tree/master/custom-lambda-authorizer). 
+
+2. In AWS IAM configure an execution role for the lambda function.
+
+    I. Navigate to IAM > roles in AWS console and create a new role with below details.
```

Comment:
Let's use simple roman numbers to keep consistency.

Reply:
fixed with https://github.com/wso2/docs-apim/pull/9205/commits/00f3cc35ed707d8ac8ae03fd09f65e7091116e12